src/apps/products/services/product_services.py

```diff
@@ -70,7 +70,7 @@ def update_single_product(session: Session, product_input: ProductInputSchema, p
         session.execute(insert(association_table).values(rows))
 
     product_data.pop('categories_ids')
-    statement = update(Product).filter(Product.id==product_id).values(**product_data)
+    statement = update(Product).filter(Product.id==product_id).values(**product_data(exclude_unset=True))
```

Comment:
Nie pamietam czy produkt ma jakies nullowalne pola, jezeli tak to ta flaga uniemozliwi setowanie pol z powrotem na nulla. Jezeli sa required to jest spoko

Reply:
Wszystkie są required